deps/cares/src/ares_process.c

```diff
@@ -60,6 +60,9 @@
 #include "ares_nowarn.h"
 #include "ares_private.h"
 
+#ifndef T_OPT
+#  define T_OPT  41 /* EDNS0 option (meta-RR) */
+#endif
```

Comment:
When this PR is landed, should this be landed as a separate commit that can be floated as a patch for subsequent updates?

Reply:
This piece of code is used in other places of c-ares, too. I'm going to provide a pull request for it in c-ares. Don't know how this should be handled in nodejs.

Reply:
Here it is: https://github.com/c-ares/c-ares/pull/385